MonoGame.Framework/Platform/SDL/SDL2.cs

```diff
@@ -631,6 +639,21 @@ public struct MotionEvent
             public int Yrel;
         }
 
+        [StructLayout(LayoutKind.Sequential)]
+        public struct ButtonEvent
+        {
+            public EventType Type;
+            public uint TimeStamp;
+            public uint WindowId;
+            public uint Which;
+            public Button Button;
```

Comment:
This button is not the same as the button flags returned from SDL_GetMouseState. It is one of a button ordinal defined in SDL_mouse.h in the SDL source code.
`public ButtonOrdinal Button`